libs/sdk-ui-kit/src/Header/HeaderHelp.tsx

```diff
@@ -0,0 +1,115 @@
+// (C) 2021 GoodData Corporation
+import React, { useState } from "react";
+import { FormattedMessage, injectIntl, IntlShape } from "react-intl";
+import { Overlay } from "../Overlay";
+import cx from "classnames";
+import isEmpty from "lodash/isEmpty";
+
+interface itemPropType {
+    key: string;
+    href?: string;
+    isActive?: boolean;
+    className?: string;
+    target?: string;
+    onClick?: (obj: any) => void;
+}
+
+interface IHeaderHelpProps {
+    intl: IntlShape;
+    className: string;
+    items: itemPropType[];
+    onMenuItemClick?: (...args: any[]) => void;
+    disableDropdown?: boolean;
+    onHelpClicked?: (isOpen?: boolean) => void;
+    redirectUrl?: string;
+}
+
+export const CoreHeaderHelp: React.FC<IHeaderHelpProps> = ({
+    className,
+    items,
+    onMenuItemClick,
+    disableDropdown,
+    onHelpClicked,
+    redirectUrl,
+}) => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    const getClassNames = () => {
```

Comment:
This does not need to be a function, you can do `const classNames = ...`. Similarly for `getMenuItems`.